open-apparel-registry/open-apparel-registry
# Update facility details content UX

## Overview

Update the UX for the facility details page:
- Adds the new "Report" dropdown button to contain the existing links that appear at the bottom of the facility profile
- Adds a "drawer' component that opens on the right and shows multiple values for a field instead of an expanding list.
- Updates the color/styling

[Wireframes](https://www.figma.com/file/FGVrJYElirWCQyR7nBehF0/OAR?node-id=468%3A16001)

Connects #2075 

## Demo

Closed Facility
<img width="524" alt="Screen Shot 2022-09-08 at 10 52 54 AM" src="https://user-images.githubusercontent.com/21046714/189156518-ba4e8262-c35d-49ea-a00d-92f2dc24f443.png">
<img width="1404" alt="Screen Shot 2022-09-08 at 10 54 21 AM" src="https://user-images.githubusercontent.com/21046714/189156582-baed26d4-9c56-4390-9271-201d46b14a74.png">

Claimed facility
<img width="708" alt="Screen Shot 2022-09-08 at 10 53 30 AM" src="https://user-images.githubusercontent.com/21046714/189156659-02173667-7a76-4629-bfc1-ee2ffa7e6ec8.png">
<img width="1400" alt="Screen Shot 2022-09-08 at 10 53 50 AM" src="https://user-images.githubusercontent.com/21046714/189156735-cb107533-3a11-422c-842e-b84d94c51925.png">

Multiple-contributor facility
<img width="612" alt="Screen Shot 2022-09-08 at 10 53 18 AM" src="https://user-images.githubusercontent.com/21046714/189156805-1125b0b7-5d0c-4661-aedf-925740e0f3f2.png">
<img width="1407" alt="Screen Shot 2022-09-08 at 10 54 02 AM" src="https://user-images.githubusercontent.com/21046714/189156823-33d6d629-8841-4545-8873-c171b34c45fb.png">

Embedded map
<img width="1404" alt="Screen Shot 2022-09-08 at 10 56 53 AM" src="https://user-images.githubusercontent.com/21046714/189156872-5dfa12a5-4a64-4321-afb6-546854e15e4b.png">

## Notes

Out of scope:
- "last updated"
- Interactive map

## Testing Instructions

* Navigate to any facility and confirm the general appearance matches the wireframes
* Test each of the actions in the 'Report' dropdown, as well as the 'Copy link' button
* Using the 'Report' dropdown, report the facility as closed, then log in as user c1@example.com and approve the closure [here](http://localhost:6543/dashboard/activityreports).
* Return to the same facility and confirm the closure flag and 'Status' details reports match the rest of the UI
* [Claim a facility](http://localhost:6543/dashboard/claims), then view the facility details. It should have a 'pending' claim flag. 
* Approve the claim as c1@example.com
* As the claimant, go to My Facilities and fill in the data for the claimed facility, then view the facility details. Confirm the claimed details are shown and match the rest of the UI. The 'claimed' flag at the top of the facility should show it has been claimed. The 'report' dropdown should now have 'dispute claim' listed as an option.
* Find a facility with at least two contributors and view its details. 
* Open the 'more contributions' drawers and ensure they match the wireframes. 
* Open the 'contributors' drawer and ensure you can follow the linked contributor names to the contributor profiles.

## Checklist
